settings/settings.json

```
@@ -1,615 +1,622 @@
+            "description": "The secondary text color for all screens in Jellyfin.",
+            "settingName": "colorSubText",
+            "type": "colorgrid",
+            "default": "#777777FF"
+          },
+          {
+            "title": "Cursor",
+            "description": "The highlight color for the cursor showing where the user's focus is.",
+            "settingName": "colorCursor",
+            "type": "colorgrid",
+            "default": "#FF6666"
+          },
+          {
+            "title": "What's New Author",
+            "description": "The text color for the author in the What's New popup.",
+            "settingName": "colorWhatsNewAuthor",
+            "type": "colorgrid",
+            "default": "#FF6666"
+          },
+          {
+            "title": "Watched Check Mark & Unplayed Count",
+            "description": "Watched Check Mark & Unplayed Count color settings",
+            "children": [
+              {
+                "title": "Background",
+                "description": "The background color of the check mark & unplayed count shown in the top right corner of items.",
+                "settingName": "colorPlayedCheckmarkBackground",
+                "type": "colorgrid",
+                "default": "#6666FF"
+              },
+              {
+                "title": "Watched Check Mark",
+                "description": "The color of the watched check mark.",
+                "settingName": "colorPlayedCheckmarkIcon",
+                "type": "colorgrid",
+                "default": "#FFFFFF"
+              },
+              {
+                "title": "Unplayed Count",
+                "description": "The text color of the unplayed count number.",
+                "settingName": "colorUnplayedCountTextColor",
+                "type": "colorgrid",
+                "default": "#FFFFFF"
+              }
+            ]
+          }
         ]
-    },
-    {
-        "title": "Global",
-        "description": "Global settings that affect everyone that uses this Roku device.",
+      }
+    ]
+  },
+  {
+    "title": "Global",
+    "description": "Global settings that affect everyone that uses this Roku device.",
+    "children": [
+      {
+        "title": "Remember Me?",
+        "description": "Remember the currently logged in user and try to log them in again next time you start Jellyfin.",
+        "settingName": "global.rememberme",
+        "type": "bool",
+        "default": "true"
+      }
+    ]
+  },
+  {
+    "title": "Playback",
+    "description": "Settings relating to playback and supported codec and media types.",
+    "children": [
+      {
+        "title": "Autoplay Episode Limit",
+        "description": "The number of episodes that will play automatically before stopping. Requires 'Play next episode automatically' server setting to be enabled. Set to 0 to disable.",
+        "settingName": "playback.episodeQueueLimit",
+        "type": "integer",
+        "default": "0"
```

Isn't the current, hard coded, default value 50?

We should just set this default 50 and remove the hard coded value in the task.

```suggestion
        "default": "50"
```